.env.docker

```
@@ -529,6 +529,7 @@ MCP_CORS_CREDENTIALS=true
 MCP_CORS_ORIGIN=http://localhost:3000,http://localhost:4200,http://127.0.0.1:3000,http://127.0.0.1:4200
 MCP_HTTP_HOST=0.0.0.0
 MCP_HTTP_PORT=3001
+MCP_AUTH_PORT=3003
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MCP_AUTH_PORT key should go before the MCP_CORS_CREDENTIALS key

(UnorderedKey)